libraries/AP_HAL_SITL/sitl_gps.cpp

```diff
@@ -475,7 +475,7 @@ void SITL_State::_update_gps_mtk16(const struct gps_data *d)
     struct timeval tv;
 
     simulation_timeval(&tv);
-    tm = *gmtime(&tv.tv_sec);
+    gmtime_r(&tv.tv_sec, &tm);
```

Comment:
If we are single threaded, then it shouldn't have problem. it was a recommanded change